x/fswap/types/genesis.go

```diff
@@ -1,24 +1,18 @@
 package types
 
-import (
-// this line is used by starport scaffolding # genesis/types/import
-)
-
-// DefaultIndex is the default capability global index
-const DefaultIndex uint64 = 1
-
 // DefaultGenesis returns the default Capability genesis state
 func DefaultGenesis() *GenesisState {
 	return &GenesisState{
-		// this line is used by starport scaffolding # genesis/types/default
-		Params: DefaultParams(),
+		Swapped: DefaultSwapped(),
```

Comment:
The returned value would depend on the literal via `DefaultConfig()`. The default configuration itself would be OK, because the application may modify the config in its initialization. However, the content of `DefaultGenesis()` would NOT be affected by the injected configuration. So you should modify the signature of `DefaultGenesis()`.